core/experiments/manager.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+namespace Elementor\Core\experiments;
+
+use Elementor\Core\Base\Base_Object;
+use Elementor\Tools;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Manager extends Base_Object {
+
+	const STATUS_ALPHA = 1;
+
+	const STATUS_BETA = 2;
+
+	const STATE_DEFAULT = 0;
+
+	const STATE_ACTIVE = 1;
+
+	const STATE_INACTIVE = 2;
```

Comment:
Please use strings for easier debug/explore db.

Reply:
Done